packages/create-react-app/index.js

```diff
@@ -153,7 +153,9 @@ function getInstallPackage(version) {
 // Extract package name from tarball url or path.
 function getPackageName(installPackage) {
   if (installPackage.indexOf('.tgz') > -1) {
-    return installPackage.match(/^.+\/(.+)-.+\.tgz$/)[1];
+    // The package name could be with or without semver version, e.g. react-scripts-0.2.0-alpha.1.tgz
+    // However, This function returns package name only wihout semver version.
+    return installPackage.match(/^.+\/(.+)-\d+.+\.tgz$/)[1];
```

Comment:
Code changed.

Regex breakdown:

* `^.+\/` starts with any characters until last slash
  * `(.+?)` searches for package name (non-greedy for searching optional semver version) and captures it in the group 1
  * `(?:-\d+.+)?` (non-capturing group) searches for any version starts with `-\d+`
* \.tgz$ finally, the tarball file extension